vnext/Microsoft.ReactNative/DesktopWindowMessage.idl

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReactNative {
+  [default_interface][webhosthidden] runtimeclass DesktopWindowMessage {
+    UInt32 uMsg {
```

Comment:
>uMsg [](start = 11, length = 4)

Please follow the Microsoft guidelines for naming in the public APIs. It must not use the Hungarian notation and use the full names without abbreviations. See [https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions)

Reply:
This is aligning to the window message API, we want people to be able to find things with the same name and not come up with new names to explain existing concepts.

Reply:
Can we use the same names as in the existing API that follows the Microsoft naming guidelines: https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.message?view=net-5.0
The Message struct plays the same role there.